server/datastore/mysql/microsoft_mdm.go

```diff
@@ -879,6 +879,17 @@ SELECT
         ELSE
             'verifying'
         END)
+    WHEN 'profiles_verified' THEN (
+        CASE (%s)
+        WHEN 'bitlocker_failed' THEN
+            'failed'
+        WHEN 'bitlocker_pending' THEN
+            'pending'
+        WHEN 'bitlocker_verifying' THEN
+            'verifying'
+        ELSE
+            'verified'
+        END)
```

Comment:
Added to mirror the hosts filter. This additional case isn't strictly needed here because unlike the host filter this particular query is only used for the summary when we expect to have a bitlocker status (so we can go straight to the final else in this particular case), but I went ahead and included it here for consistency and future clarity.